src/main/java/com/manardenza/dao/HotelDaoImpl.java

```diff
@@ -0,0 +1,25 @@
+package com.manardenza.dao;
+
+import com.manardenza.entity.Hotel;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HotelDaoImpl extends AbstractDao<Hotel> {
+
+    private static HotelDaoImpl instance;
+
+    List<Hotel> hotelList = new ArrayList<>();
```

Comment:
List<T> may be stored inside abstract dao. Make it protected, so all children will have access to it